workspaces/src/rpc/client.rs

```diff
@@ -72,13 +73,12 @@ impl Client {
         .await
     }
 
-    // TODO: return a type T instead of Value
```

Comment:
Unrelated to the PR, but in the `call` method, I'd made all parameters required, and left defaults to the builder. It's better to set defaults once, at the outer layer.